src/main/java/seedu/weeblingo/logic/parser/StartCommandParser.java

```diff
@@ -29,8 +29,9 @@ public StartCommand parse(String args) throws ParseException {
         } else {
             try {
                 numberOfQuestions = Integer.parseInt(stringNumberOfQuestions.get());
+                System.out.println(numberOfQuestions);
```

Comment:
Probably shouldn't be leaving debugging lines in our actual code.

Reply:
thanks for spotting. removed